### ISSUE TITLE
[notf] dtypes.py is setting wrong _np_bfloat16

### ISSUE THREAD

**Issue by stephanwlee** [tensorflow/tensorboard#1677]
np_bloat16 in no TF mode is set to `0`.
https://github.com/tensorflow/tensorboard/blob/51e9611843caea57bebdc1d8cf2a2c9c1ecf090b/tensorboard/compat/tensorflow_stub/pywrap_tensorflow.py#L39

That causes lines like these fail:
https://github.com/tensorflow/tensorboard/blob/51e9611843caea57bebdc1d8cf2a2c9c1ecf090b/tensorboard/compat/tensorflow_stub/dtypes.py#L208
https://github.com/tensorflow/tensorboard/blob/51e9611843caea57bebdc1d8cf2a2c9c1ecf090b/tensorboard/compat/tensorflow_stub/dtypes.py#L679-L681

**Comment by nfelt** [comment]
Yes, we need a better fix for this.  The real answer IMO is just eliminating the use of dtypes from the stub as much as possible since I don't think we need them except for pretty trivial stuff.

**Comment by shashvatshahi1998** [comment]
@stephanwlee can you please guide me regarding this, I want to work on this issue.
